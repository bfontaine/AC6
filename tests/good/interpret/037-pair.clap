val fst = case { { _1 -> x, _2 -> _ } => x }
val snd = case { { _1 -> _, _2 -> x } => x }
val p = { _1 <- 42, _2 <- 0 }
val z = fst p
val y = snd p

