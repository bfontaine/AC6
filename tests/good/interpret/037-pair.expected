:- fst = <code>
:- snd = <code>
:- p = {_1 <- 42, _2 <- 0 }
:- z = 42
:- y = 0
